packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx

```diff
@@ -144,7 +146,9 @@ function CustomViewLoader(props: TCustomViewLoaderProps) {
         key={`custom-view-${props.customView.id}`}
         ref={iFrameElementRef}
         title={`Custom View: ${props.customView.defaultLabel}`}
-        src={props.customView.url}
+        src={`${window.location.origin}/${
```

Comment:
iFrame URL will be a MC one since we want to proxy the request, the same way we do for Custom Applications:
```
https://<MC_HOST>/<project_key>/custom-view/<custom_view_id>
```

This URL will not be visible for users.
It has a fixed path so we can treat it specially in the `http-proxy`.

Reply:
If there is a Custom Application with the entry point `custom-view`, we might have a conflict / issue here.

Since this will be a route served through the MC Proxy, maybe we can use another (more unique) path?

Some examples:

```
<MC_HOST>/custom-views/<project_key>/<custom_view_id>
<MC_HOST>/frames/<project_key>/custom-views/<custom_view_id>
<MC_HOST>/custom-views/<custom_view_id>?project-key=<project_key>
```

What do you think? Essentially an endpoint that does not start with `<project_key>`.